packages/ui-top-nav-bar/src/TopNavBar/TopNavBarLayout/DesktopLayout/index.tsx

```diff
@@ -51,7 +51,7 @@ private: true
 @deprecated(
   '9',
   { hideActionsUserSeparator: true },
-  'From v9, actionUserSeparator has to be visible at all times. Please do not use designs which hide it. Note: this prop is probably used inside the desktopConfig prop on [TopNavBar.Layout]'
+  'further info: https://google.com'
```

Comment:
I love using Google when I have a concern about something but is this really the best thing to recommend here?